docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -140,15 +148,74 @@ components.
 
 ## Component health
 
-`prometheus.scrape` will only be reported as unhealthy when given an invalid
+`prometheus.scrape` is only reported as unhealthy if given an invalid
 configuration.
 
 ## Debug information
 
 `prometheus.scrape` reports the status of the last scrape for each configured
 scrape job on the component's debug endpoint.
 
-### Debug metrics
+## Debug metrics
 
 `prometheus.scrape` does not expose any component-specific debug metrics.
 
+## Scraping behavior
+The `prometheus.scrape` component borrows the scraping behavior of Prometheus.
+Prometheus, and by extent this component, uses a pull model for scraping
+metrics from a given set of _targets_.
+Each scrape target is defined as a set of key-value pairs called _labels_.
+The set of targets can either be _static_, or dynamically provided periodically
+by a service disovery component such as `discovery.kubernetes`. The special
+label `__address__` _must always_ be present and corresponds to the
+`<host>:<port>` that is used for the scrape request.
+
+By default, the scrape job tries to scrape all available targets' `/metrics`
+endpoints using HTTP, with a scrape interval of 1 minute and scrape timeout of
+10 seconds. The metrics path, protocol scheme, scrape interval and timeout,
+query parameters, as well as any other settings can be configured using the
+component's arguments.
+
+The scrape job expects the metrics exposed by the endpoint to follow the
+[OpenMetrics](https://openmetrics.io/) format. All metrics are then propagated
+to each receiver listed in the component's `forward_to` argument.
+
+Labels coming from targets, that start with a double underscore `__` are
+treated as _internal_, and are removed prior to scraping.
+
+The `prometheus.scrape` component regards a scrape as successful if it
+responded with a 200 status code and returned a body of valid metrics.
+
+If the scrape request fails, the component's debug endpoint and debug UI
+section contains more detailed information about the failure, the last
+successful scrape, as well as the labels last used for scraping.
```

Comment:
We're going to end up suggesting just the UI since it scales better, and I'll probably want to remove the existing debug endpoint:

```suggestion
If the scrape request fails, the component's debug UI
section contains more detailed information about the failure, the last
successful scrape, as well as the labels last used for scraping.
```